<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidSoftUI</class>
 <widget class="QDialog" name="MidSoftUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MidSoftUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="minsoft.qrc">
    <normaloff>:/power_32.png</normaloff>:/power_32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>器件及地址配置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,2,1">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="title">
           <string>Simulation Software(仿真软件)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <family>Agency FB</family>
                 <pointsize>9</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>器件类型：          </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="styleSheet">
                <string notr="true">QComboBox{min-height:23;}</string>
               </property>
               <item>
                <property name="text">
                 <string>电力软件</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>通信软件</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>电力应用</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>中间件IP：  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="styleSheet">
                <string notr="true">QLineEdit{min-height:23;}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>中间件Port：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="styleSheet">
                <string notr="true">QLineEdit{min-height:23;}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>SBS(软总线)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>  IP： </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit{min-height:23;}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Port： </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="styleSheet">
                <string notr="true">QLineEdit{min-height:23;}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>通信方式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">QComboBox{min-height:23;}</string>
            </property>
            <item>
             <property name="text">
              <string>tcp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>udp</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>进程信息</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2,1,2,1">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="font">
        <font>
         <family>隶书</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="unregisterButton">
       <property name="font">
        <font>
         <family>隶书</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="minsoft.qrc"/>
 </resources>
 <connections/>
</ui>
